lib/temp-directory.js

```diff
@@ -2,21 +2,38 @@
 var path = require('path');
 
 // npm modules
+var async = require('async');
 var mkdirp = require('mkdirp');
 var rimraf = require('rimraf');
 var uuid = require('uuid');
 var osenv = require('osenv');
 
 function create(context, next) {
   var _path = path.join(osenv.tmpdir(),uuid.v4());
-  var npmConfigTmp = path.join(_path, 'npm_config_tmp');
-
-  context.path = _path;
+  var npmConfigTmp = _path;
```

Comment:
no need to set this, it is potentially confusing. perhaps just leave it as `var npmConfigTmp`